Makefile

```diff
@@ -233,14 +233,25 @@ install-resources:
 e2e-dependencies:
 
 K8SCLIENT ?= oc
+GINKGO = $(LOCAL_BIN)/ginkgo
+IS_HOSTED ?= false
+TIMEOUT_SECONDS=30
+MANAGED_CLUSTER_NAMESPACE ?= cluster2-hosted
+
 .PHONY: e2e-test
 e2e-test:
 	@if [ -z "$(TEST_FILE)" ]; then\
-		$(GINKGO) -v --no-color $(TEST_ARGS) --fail-fast test/e2e -- -cluster_namespace=$(MANAGED_CLUSTER_NAME) -k8s_client=$(K8SCLIENT) ;\
+		$(GINKGO) -v --no-color $(TEST_ARGS) --fail-fast test/e2e -- -cluster_namespace=$(MANAGED_CLUSTER_NAMESPACE) -k8s_client=$(K8SCLIENT) -is_hosted=$(IS_HOSTED) -cluster_namespace_on_hub=$(CLUSTER_NAMESPACE_ON_HUB) -timeout_seconds=$(TIMEOUT_SECONDS);\
```

Comment:
What fails with the reduced timeout in hosted mode? I think it might be because the controllers are installed asynchronously by the addon controller in hosted mode whereas in normal mode, the Makefile waits for the pods to be up.

I think if you add to the Makefile to wait for the controllers to be running on the hosting cluster, you might not need this additional timeout increase.

Reply:
what is suggestion? it is not failed during the installation. it is failed during cert and iam e2e test

Reply:
let me take a screen shot for u